packages/combobox/src/LionCombobox.js

```diff
@@ -218,8 +272,10 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
    * @protected
    */
   get _listboxNode() {
-    return /** @type {LionOptions} */ ((this._overlayCtrl && this._overlayCtrl.contentNode) ||
-      Array.from(this.children).find(child => child.slot === 'listbox'));
+    return /** @type {LionOptions} */ (
```

Comment:
I think you should do a commit with the correct prettier (remove node_modules and do a fresh install based on lock file of that time)